apps/comps/components/timeline-chart/calmar-ratio-chart.tsx

```diff
@@ -69,22 +69,12 @@ export const CalmarRatioChart: React.FC<CalmarRatioChartProps> = ({
   const chartDisplayAgents = useMemo(() => {
     if (!debouncedSearchQuery) {
       // When not searching, use pagination from parent component
-      const startIndex = (currentPage - 1) * LIMIT_AGENTS_PER_PAGE;
-      const endIndex = startIndex + LIMIT_AGENTS_PER_PAGE;
-      return agentsWithCalmar.slice(startIndex, endIndex);
```

Comment:
We can't slice because `agentsWithCalmar` never has more than the server's page length.